.github/workflows/ci.yml

```diff
@@ -141,13 +136,6 @@ jobs:
           --out-dir $GITHUB_WORKSPACE/build/output
           cmake --build $GITHUB_WORKSPACE/build --parallel 2
 
-      - name: Clang Tidy
```

Comment:
We can also remove `-DCMAKE_EXPORT_COMPILE_COMMANDS=ON` option in `Configure CMake` step